.github/bin/deploy-run-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+BASIC_AUTH_BASE64=""
+
+## Prevent errors when basic auth isn't used
+##
+## Nb.the BASIC_AUTH_USER secret in GH production environment should
+## be set to `no-basic-auth` if not being used
+if [ "$BASIC_AUTH_USER" != "no-basic-auth" ]; then
```

Comment:
Really nice that this must be explicitly set.

I'd like to have a table in the readme at some point for the env vars. That would be a good place to make a note about how this works :)